nbbuild/javadoctools/template.xml

```diff
@@ -279,10 +279,6 @@ cause it to fail.
             <replacefilter token="@nb-docs-css@" value="${nb-docs.css}"/>
             <replacefilter token="@api-stability-image@" value="${stability.image}"/>
         </replace>
-        <mkdir dir="${javadoc.out.dir}/resources"/>
-        <mkdir dir="${javadoc.out.dir}/resource-files/resources"/>
-        <copy file="${javadoc.out.dir}/resource-files/glass.png" todir="${javadoc.out.dir}/resource-files/resources" />
-        <copy file="${javadoc.out.dir}/resource-files/x.png" todir="${javadoc.out.dir}/resource-files/resources" />
```

Comment:
@ebarboni  JDK 25 changed the two PNGs into SVGs. I am not sure why those icons were copied in https://github.com/apache/netbeans/pull/8368. I suppose if i change it to copy the svg some path or css would need updating?

Reply:
updated the PR, it copies now the SVGs on JDK 25.